php/commands/sites.php

```diff
@@ -162,37 +162,46 @@ public function create($args, $assoc_args) {
   public function import($args, $assoc_args) {
     $options = Sites_Command::getSiteCreateOptions($assoc_args);
 
-    $url = Input::string($assoc_args, 'url', "URL of archive to import");
+    $url = Input::string($assoc_args, 'url', 'URL of archive to import');
     if (!$url) {
-      Terminus::error("Please enter a URL.");
+      $this->logger->error('Please enter a URL.');
     }
 
-    $workflow = $this->sites->addSite($options);
+    try {
```

Comment:
This feels like a slightly non-standard use of try/catch. It's appropriate in this case but we should probably add a comment here explaining it to help new developers grok the flow it a bit more quickly.